src/containers/Settings/components/Security/index.js

```diff
@@ -285,43 +285,43 @@ class Security extends Component {
                                 {I18n.t('commons.security')}
                             </h2>
                             
-                            <div className="listElement">
-                                <div className="message">
+                            <div className="list-element">
+                                <div className="list-element__message">
                                     {I18n.t('commons.password')}
-                                    <div className="detail">
+                                    <div className="list-element__detail">
                                         {I18n.t('settings.security.change_flyve_passsword')}
                                     </div>
                                 </div>
-                                <div className="controller">
-                                    <button className="btn --secondary" onClick={() => this.changeMode(I18n.t('commons.change_password'))}>
+                                <div className="list-element__controller">
+                                    <button className="btn btn--secondaryaryary" onClick={() => this.changeMode(I18n.t('commons.change_password'))}>
                                         {I18n.t('commons.edit')}
                                     </button>
                                 </div>
                             </div>
             
-                            <div className="listElement">
-                                <div className="message">
+                            <div className="list-element">
+                                <div className="list-element__message">
                                     {I18n.t('commons.kill_session')}
-                                    <div className="detail">
+                                    <div className="list-element__detail">
                                         {I18n.t('settings.security.destroy_session')}
                                     </div>
                                 </div>
-                                <div className="controller">
-                                    <button className="btn --secondary" onClick={this.closeSession}>
+                                <div className="list-element__controller">
+                                    <button className="btn btn--secondaryaryary" onClick={this.closeSession}>
```

Comment:
@Gianfranco97 look at this:
- primaryryry
- secondaryaryary
lol

Reply:
Thanks, I'm going to fix that 👍